modules/data-table/main/index.coffee

```diff
@@ -786,10 +786,18 @@ urlFeed = (url, options) ->
 
   {
     url: url # for debugging
-    headers: maybeCached (r) -> hx.json(url, { type: 'headers', extra: options.extra }, r)
-    totalCount: maybeCached (r) -> hx.json(url, { type: 'totalCount', extra: options.extra }, (res) -> r(res.count))
-    rows: (range, cb) -> hx.json(url, { type: 'rows', range: range, extra: options.extra }, cb)
-    rowsForIds: (ids, lookupRow, cb) -> hx.json(url, { type: 'rowsForIds', ids: ids, extra: options.extra }, cb)
+    headers: maybeCached (cb) ->
+      hx.json url, { type: 'headers', extra: options.extra }, (err, res) ->
+        cb(res)
```

Comment:
I think `console.error(err)` makes the most sense in this situation in terms of what browsers do when a request has an error. If you need more advanced error handling then you can always write a custom feed.